src/components/PeopleFilters.tsx

```diff
@@ -16,6 +43,12 @@ export const PeopleFilters = () => {
             type="search"
             className="input"
             placeholder="Search"
+            value={query}
+            onChange={(e) => setsearchParams(
+              getSearchWith(
+                searchParams, { query: e.target.value || null },
+              ),
+            )}
```

Comment:
instead anonymous arrow function better to make function in component and pass reference here